src/schemas/contact.ts

```diff
@@ -19,6 +20,11 @@ export enum ContactType {
   Personal = Individual,
 }
 
+export interface ContactType {
+  baseType: BaseType,
+  corpInternal?: boolean,
```

Comment:
I'd like to keep our payload to be flattening.

Could we try to find a way not using a nested structure so that our payload can be as simple as possible?

Thanks!

Reply:
Moved the `corpInternal` out to the root object.